baseplate/clients/cassandra.py

```diff
@@ -21,12 +22,38 @@
 from cassandra.query import BoundStatement  # pylint: disable=no-name-in-module
 from cassandra.query import PreparedStatement  # pylint: disable=no-name-in-module
 from cassandra.query import SimpleStatement  # pylint: disable=no-name-in-module
+from prometheus_client import Counter
+from prometheus_client import Gauge
+from prometheus_client import Histogram
 
 from baseplate import Span
 from baseplate.clients import ContextFactory
 from baseplate.lib import config
+from baseplate.lib.prometheus_metrics import default_latency_buckets
```

Comment:
I'm worried about the floats in this list causing buckets to not align, if we keep using it we should probably round it to 4ish digits.

Reply:
They're the same everywhere though. Why would they not align?
Or did you mean between go and python for instance?